ops/cli.py

```diff
@@ -1,351 +1,343 @@
-        )
-        parser.add_argument(
-            "-g",
-            "--gotc-folder",
-            dest="gotc_folder",
-            help="Overrides ENVIRONMENT; specify exact 'gotc-deploy/deploy/{GOTC_FOLDER}' path to use."
-        )
-        parser.add_argument(
-            "-n",
-            "--namespace",
-            dest="namespace",
-            default="default",
-            help="Deploy to this namespace."
-        )
-        parser.add_argument(
-            "-d",
-            "--dry-run",
-            dest="dry_run",
-            action="store_true",
-            help="When provided, exit after printing parsed arguments."
-        )
-        parser.add_argument(
-            "--cluster-no-zone-name",
-            dest="cluster_no_zone_name",
-            action="store_true",
-            help="When provided, do not append '-{ZONE}' to the CLUSTER name."
-        )
-        args = parser.parse_args(arguments)
-        project = args.project if args.project is not None else f"broad-{args.environment}"
-        cluster = args.cluster if args.cluster is not None else f"{args.environment}-shared"
-        if not args.cluster_no_zone_name:
-            cluster = f"{cluster}-{args.zone}"
-        gotc_folder = args.gotc_folder if args.gotc_folder is not None else args.environment
-        info("=> Deploy config:")
-        info(f"   GCP project: {project}")
-        info(f"   GCP zone: {args.zone}")
-        info(f"   GCP cluster: {cluster} (namespace: {args.namespace})")
-        info(f"   GOTC deploy folder: gotc-deploy/deploy/{gotc_folder}")
-        if args.dry_run:
-            info("=> Exiting due to --dry-run")
-            return 0
+    def run_cloudsql_proxy(self):
+        """Use docker via gce-proxy to connect to the Cloud SQL instance."""
+        info("=>  Running cloud_sql_proxy")
+        token = shell("gcloud auth print-access-token")
+        connectionName = json.loads(shell(f"gcloud --project {self.project} --format=json sql instances"
+                                          f"describe {self.cloudsql_name}"))["connectionName"]
+        return shell(f"docker run --rm -d -p 127.0.0.1:5432:5432 gcr.io/cloudsql-docker/gce-proxy:1.16 "
+                     f"/cloud_sql_proxy -token='{token}' -instances='{connectionName}=tcp:0.0.0.0:5432'")
+
 
-        commands_available()
-        set_up_helm()
-        set_up_k8s(project=project, namespace=args.namespace, cluster=cluster, zone=args.zone)
+class DeployCommand(ConnectCommand):
+    """Class for deploy command that deploys WFL to infrastructure."""
 
-        version = shell("cat version")
-        publish_docker_images(version=version)
+    def __init__(self, parsed_args: argparse.Namespace):
+        super().__init__(parsed_args)
+
+    def execute(self) -> int:
+        """Set up helm, publish docker, render CTMPL, helm deploy, migrate liquibase."""
+
+        if not input(f"Are you sure you want to deploy version {self.version}? [N/y]").lower().startswith("y"):
```

Comment:
Accessing properties of the base class like this will cause problems down the line if the base class changes.